### PR TITLE
fix: remove dns value specs

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:
Remove DNS field `value_specs`. Include resource `huaweicloud_dns_recordset` and `huaweicloud_dns_zone`.
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccDNSV2RecordSet_'
...
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud -v -run TestAccDNSV2RecordSet_ -timeout 360m -parallel 4 
=== RUN   TestAccDNSV2RecordSet_basic 
=== PAUSE TestAccDNSV2RecordSet_basic
=== RUN   TestAccDNSV2RecordSet_readTTL
=== PAUSE TestAccDNSV2RecordSet_readTTL
=== RUN   TestAccDNSV2RecordSet_private
=== PAUSE TestAccDNSV2RecordSet_private
=== CONT  TestAccDNSV2RecordSet_basic
=== CONT  TestAccDNSV2RecordSet_private
=== CONT  TestAccDNSV2RecordSet_readTTL
--- PASS: TestAccDNSV2RecordSet_readTTL (39.02s) 
--- PASS: TestAccDNSV2RecordSet_private (44.43s) 
--- PASS: TestAccDNSV2RecordSet_basic (70.31s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.410s
```

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccDNSV2Zone_'
...
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud -v -run TestAccDNSV2Zone_ -timeout 360m -parallel 4 
=== RUN   TestAccDNSV2Zone_basic 
=== PAUSE TestAccDNSV2Zone_basic
=== RUN   TestAccDNSV2Zone_private
=== PAUSE TestAccDNSV2Zone_private
=== RUN   TestAccDNSV2Zone_readTTL
=== PAUSE TestAccDNSV2Zone_readTTL
=== RUN   TestAccDNSV2Zone_withEpsId
=== PAUSE TestAccDNSV2Zone_withEpsId
=== CONT  TestAccDNSV2Zone_basic
=== CONT  TestAccDNSV2Zone_readTTL
=== CONT  TestAccDNSV2Zone_private
=== CONT  TestAccDNSV2Zone_withEpsId
--- PASS: TestAccDNSV2Zone_readTTL (26.39s) 
--- PASS: TestAccDNSV2Zone_withEpsId (27.10s) 
--- PASS: TestAccDNSV2Zone_private (27.12s)
--- PASS: TestAccDNSV2Zone_basic (44.46s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       44.514s
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_dns_recordset_v2.go [hunk 1]
```diff
@@ -101,15 +95,12 @@ func resourceDNSRecordSetV2Create(d *schema.ResourceData, meta interface{}) erro
 		records[i] = recordraw.(string)
 	}
 
-	createOpts := RecordSetCreateOpts{
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_dns_recordset_v2.go:104]: the define of `RecordSetCreateOpts` should also be deleted.

File: huaweicloud/resource_huaweicloud_dns_zone_v2.go [hunk 2]
```diff
@@ -150,19 +142,15 @@ func resourceDNSZoneV2Create(d *schema.ResourceData, meta interface{}) error {
 			return fmtp.Errorf("Error creating HuaweiCloud DNS region client: %s", err)
 		}
 	}
-	vs := MapResourceProp(d, "value_specs")
-	// Add zone_type to the list
-	vs["zone_type"] = zoneType
-	vs["router"] = resourceDNSRouter(d, region)
-	createOpts := ZoneCreateOpts{
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_dns_zone_v2.go:157]: ditto

**Comment by deer-hang** [comment]
```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccDNSV2RecordSet_'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud -v -run TestAccDNSV2RecordSet_ -timeout 360m -parallel 4 
=== RUN   TestAccDNSV2RecordSet_basic 
=== PAUSE TestAccDNSV2RecordSet_basic
=== RUN   TestAccDNSV2RecordSet_readTTL
=== PAUSE TestAccDNSV2RecordSet_readTTL
=== RUN   TestAccDNSV2RecordSet_private
=== PAUSE TestAccDNSV2RecordSet_private
=== CONT  TestAccDNSV2RecordSet_basic
=== CONT  TestAccDNSV2RecordSet_private
=== CONT  TestAccDNSV2RecordSet_readTTL
--- PASS: TestAccDNSV2RecordSet_readTTL (37.58s) 
--- PASS: TestAccDNSV2RecordSet_private (38.13s) 
--- PASS: TestAccDNSV2RecordSet_basic (61.99s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       62.044s
```

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccDNSV2Zone_'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud -v -run TestAccDNSV2Zone_ -timeout 360m -parallel 4 
=== RUN   TestAccDNSV2Zone_basic 
=== PAUSE TestAccDNSV2Zone_basic
=== RUN   TestAccDNSV2Zone_private
=== PAUSE TestAccDNSV2Zone_private
=== RUN   TestAccDNSV2Zone_readTTL
=== PAUSE TestAccDNSV2Zone_readTTL
=== RUN   TestAccDNSV2Zone_withEpsId
=== PAUSE TestAccDNSV2Zone_withEpsId
=== CONT  TestAccDNSV2Zone_basic
=== CONT  TestAccDNSV2Zone_readTTL
=== CONT  TestAccDNSV2Zone_withEpsId
=== CONT  TestAccDNSV2Zone_private
--- PASS: TestAccDNSV2Zone_readTTL (23.05s) 
--- PASS: TestAccDNSV2Zone_withEpsId (25.28s) 
--- PASS: TestAccDNSV2Zone_private (25.87s) 
--- PASS: TestAccDNSV2Zone_basic (39.94s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       39.990s
```

**Pull request merged**